docs/en/development/index.md

```diff
@@ -0,0 +1,4 @@
+---
+title: "Research and Development"
+---
+{% include 'devguide' %}
```

Comment:
ideally there would be a $lang global var that allows us to 'devguide_$lang' but i wasn't able to find this after a brief search, but it should still be possible to get this translatable. 

`{% include 'devguide_en' %}` to access a `docs/macros/includes/devguide_en` _should_ be a viable method for getting macros translated.

Reply:
Would probably work to put a key in the frontmatter like 
`lang: "en"`

then it would be `{% include '{{ lang }}/devguide' % }` or `{% include 'devguide_{{ lang }}' % }`

Not sure if includes folder allows subfolders or if you can nest the variable like that